packages/admin/cms-admin/src/preview/common/IFrameViewer.tsx

```diff
@@ -121,13 +121,14 @@ const OuterFrame = styled("div", { shouldForwardProp: (prop) => prop !== "device
             width: ${deviceConfig.outerFrame.width}px;
             height: ${deviceConfig.outerFrame.height}px;
             padding: ${deviceConfig.outerFrame.padding};
+            overflow: hidden;
```

Comment:
Since the scrolling is handled by the iframe itself, I think we could just remove the `overflow-y: auto;` above 🤔

Reply:
I was not sure about this and didn't want to touch code that didn't pertain to the problem I was fixing. But I will remove this line if you think it's not needed.